packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -1100,6 +1098,23 @@ export default {
                 await this.loadAvailableFulfilment();
                 this.availableFulfilmentTimesKey++;
             }
+        },
+
+        mapCheckoutUpdateRequest (ageVerificationOnly) {
```

Comment:
Can we rename this function to avoid the confusion between  `mapCheckoutUpdateRequest` and `mapUpdateCheckoutRequest`

Reply:
Changed. Naming variables/functions is the hardest part of the job 🙈